search/FloydAlgorithmForCycleDetection.cpp

```diff
@@ -0,0 +1,42 @@
+
+/*Given an array of integers nums containing n + 1 integers where each integer is in the range [1, n] inclusive.
+
+There is only one duplicate number in nums, return this duplicate number in O(1) space and time complexity less than O(n^2) without modifying the original array.
+
+Example-:
+    Input: 5
+           3 1 3 4 2
+    
+    Output: 3
+*/
+#include<bits/stdc++.h>
+using namespace std;

```

Comment:
as per repo standards, you cannot use namespaces globally.
using cout, cin etc directly is not allowed, instead use std::cout, std::cout